api/src/main/java/io/opencensus/trace/SpanBuilder.java

```diff
@@ -146,7 +146,7 @@
    * class MyClass {
    *   private static final Tracer tracer = Tracing.getTracer();
    *   void DoWork() {
-   *     Span span = tracer.spanBuilder(null, "MyRootSpan").startSpan();
+   *     Span span = tracer.spanBuilderWithParent(null, "MyRootSpan").startSpan();
```

Comment:
since we are changing apis, some look at style and say declare parameters like a sentence. so if method says "span with parent" then parameters spanName, parent

I think this was from an old book "clean code"